index.html

```diff
@@ -86,21 +86,19 @@
 				
 				<div class="resourceleft">
 					<ul>
-						<li><a href=""><h3>Link 1</h3></a></li>
-						<li><a href=""><h3>Link 2</h3></a></li>
-						<li><a href=""><h3>Link 3</h3></a></li>
-						<li><a href=""><h3>Link 4</h3></a></li>
-					
+						<li><a href="http://javascript.info/" target="_blank" ><h3>JavaScript Tutorials</h3></a></li>
+						<li><a href="https://trello.com/" target="_blank"><h3>Trello WorkFlow Management</h3></a></li>
+						<li><a href="https://www.w3schools.com/" target="_blank"><h3>w3 Schools</h3></a></li>
+											
```

Comment:
Be very careful with whitespace. It will inevitably lead to merge conflicts. There are a several instances of whitespace in both the html file and javascript file. 
You can change your settings in sublime to automatically trim the whitespace. Just enter this line into your custom settings.
`"trim_automatic_white_space": true,`

Also, a good way to visualize trailing whitespace is to add the Trailing Spaces package to your editor. It will highlight any whitespace in your code, and make it super easy to stay on top of! :)